package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@pusher/chatkit-client",
   "description": "Pusher Chatkit client SDK for browsers and react native",
   "main": "dist/web/chatkit.js",
-  "version": "1.13.1",
+  "version": "1.13.2-beta.1",
```

Comment:
Is this meant to be here?

Reply:
no harm in it. It'll be changed after merging when we cut a non-beta release.